src/lab1-tests.md

```diff
@@ -14,91 +14,90 @@ Here's the testing boilerplate.
     
 You can run this file with `doctest` to run the tests.
 
-Testing `show`
---------------
-
-The `show` function is supposed to print some HTML to the screen, but
-skip the tags inside.
-
-    >>> lab1.show('<body>hello</body>')
-    hello
-    >>> lab1.show('he<body>llo</body>')
-    hello
-    >>> lab1.show('he<body>l</body>lo')
-    hello
-    >>> lab1.show('he<body>l<div>l</div>o</body>')
-    hello
-
-Note that the tags do not have to match:
-
-    >>> lab1.show('he<body>l</div>lo')
-    hello
-    >>> lab1.show('he<body>l<div>l</body>o</div>')
-    hello
-
-Testing `URL`
--------------
+1.4 Telnet in Python
+--------------------
 
 Constructing a `URL` object parses a URL:
 
-    >>> lab1.URL('http://test.test/example1')
-    URL(scheme=http, host=test.test, port=80, path='/example1')
+    >>> lab1.URL('http://test.test/example1') #doctest: +ELLIPSIS
```

Comment:
What does doctest: +ELLIPSIS do?

Reply:
I guess it allows pattern matching with ellipsis in the following line?

Reply:
Yes, here it allows the port to be present or absent when printing a URL.